examples/test_util/EclFile.hpp

```diff
@@ -34,7 +34,7 @@ class EclFile
 
 public:
     
-    EclFile(std::string filename);
+    explicit EclFile(std::string filename);
```

Comment:
When touching this code I guess the argument type should be changed to `const std::string&`

Reply:
done.